spec/unit/typesafe/config/path_spec.rb

```diff
@@ -0,0 +1,259 @@
+require 'spec_helper'
+require 'hocon'
+require 'test_utils'
+
+
+describe Hocon::Impl::Path do
+  Path = Hocon::Impl::Path
+
+  ####################
+  # Path Equality
+  ####################
+  context "Check path equality" do
+    # note: foo.bar is a single key here
+    let(:key_a) { Path.new_key("foo.bar") }
+    let(:same_as_key_a) { Path.new_key("foo.bar") }
+    let(:different_key) { Path.new_key("hello") }
+
+    # Here foo.bar is two elements
+    let(:two_elements) { Path.new_path("foo.bar") }
+    let(:same_as_two_elements) { Path.new_path("foo.bar") }
+
+    context "key_a equals a path of the same name" do
+      let(:first_object) { key_a }
+      let(:second_object) { TestUtils.path("foo.bar") }
```

Comment:
I think here and in the examples below within the `Check path equality` contextt, you should use the values you defined above rather than calling `TestUtils.path`.